### DISCUSSION TITLE
apply TTA during inference using sahi

### DISCUSSION THREAD

**Discussion by ramdhan1989** [obss/sahi#633] [Q&A] [answered]
Is there any example code to apply TTA? I only found an example for yolov5 using torch.hub.load. However, I could not find it for other frameworks (detectron, mmdetection, etc)

**Comment by fcakyon** [comment]
Hello @ramdhan1989, can you provide a reproducible code example so that we see which functions and modules are called?

**Comment by ramdhan1989** [comment]
I apologize, my previous question seems not related with this repo. I changed my question if that would be ok.

thanks

**Comment by fcakyon** [comment]
You can find TTA examples for Detectron2, MMDetection, HuggingFace and Torchvision models here: https://github.com/obss/sahi/tree/main/demo

**Comment by ramdhan1989** [comment]
I have checked the notebooks. However, I couldn't find the syntax that used to activate or deactivate TTA during inference. would you mind elaborating more about the TTA option in the code?

thanks

**Comment by fcakyon** [comment, marked as answer]
What do you mean by TTA?

By default, the SAHI predict function combines standard prediction and sliced prediction results.

Set `no_standard_prediction: bool = True` if you want to disable standard prediction https://github.com/obss/sahi/blob/287accf163fc10a950684e2e72eaccd12c526bf9/sahi/predict.py#L302

Set `no_sliced_prediction: bool = True` if you want to disable sliced prediction https://github.com/obss/sahi/blob/287accf163fc10a950684e2e72eaccd12c526bf9/sahi/predict.py#L303

Similarly,

Set `perform_standard_pred: bool = False` if you want to disable standard prediction in `get_sliced_prediction` function
https://github.com/obss/sahi/blob/287accf163fc10a950684e2e72eaccd12c526bf9/sahi/predict.py#L130

**Comment by ramdhan1989** [comment]
Ok noted
thanks for your information
